tests/common.go

```diff
@@ -3157,3 +3181,173 @@ func init() {
 	logrus.StandardLogger().Hooks.Add(log.NewHook())
 	logrus.SetOutput(os.Stdout)
 }
+
+//CreateJiraIssueWithLogs creates a jira issue and copy logs to nfs mount
+func CreateJiraIssueWithLogs(issueDescription, issueSummary string) {
+	issueKey := jirautils.CreateIssue(issueDescription, issueSummary)
+	if issueKey != "" {
+		collectAndCopyDiagsOnWorkerNodes(issueKey)
+		collectAndCopyStorkLogs(issueKey)
+		collectAndCopyOperatorLogs(issueKey)
+		collectAndCopyAutopilotLogs(issueKey)
+
+	}
+
+}
+
+func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
+	isIssueDirCreated := false
+	for _, currNode := range node.GetWorkerNodes() {
+		err := runCmd("pwd", currNode)
+		if err == nil {
+			logrus.Infof("Creating directors logs in the node %v", currNode.Name)
+			runCmd("mkdir /root/logs", currNode)
+			logrus.Info("Mounting nfs diags directory")
+			runCmd("mount -t nfs diags.pwx.dev.purestorage.com:/var/lib/osd/pxns/688230076034934618 /root/logs", currNode)
+			if !isIssueDirCreated {
+				logrus.Infof("Creating PTX %v directory in the node %v", issueKey, currNode.Name)
+				runCmd(fmt.Sprintf("mkdir /root/logs/%v", issueKey), currNode)
+				isIssueDirCreated = true
+			}
+
+			logrus.Infof("collect diags on node: %s", currNode.Name)
+
+			filePath := fmt.Sprintf("/var/cores/diags-%s-%d.tar.gz", currNode.Name, time.Now().Unix())
+
+			config := &torpedovolume.DiagRequestConfig{
+				DockerHost:    "unix:///var/run/docker.sock",
+				OutputFile:    filePath,
+				ContainerName: "",
+				Profile:       false,
+				Live:          true,
+				Upload:        false,
+				All:           true,
+				Force:         true,
+				OnHost:        true,
+				Extra:         false,
+			}
+			err = Inst().V.CollectDiags(currNode, config, torpedovolume.DiagOps{Validate: false, Async: true})
+
+			if err == nil {
+				filePath = fmt.Sprintf("/var/cores/%s-diags-*.tar.gz", currNode.Name)
+				logrus.Infof("copying logs %v  on node: %s", filePath, currNode.Name)
+				runCmd(fmt.Sprintf("cp %v /root/logs/%v/", filePath, issueKey), currNode)
+			} else {
+				logrus.Warnf("Error collecting diags on node: %v, Error: %v", currNode.Name, err)
+			}
+
+		}
+	}
+}
+
+func collectAndCopyStorkLogs(issueKey string) {
+
+	storkLabel := make(map[string]string)
+	storkLabel["name"] = "stork"
+	podList, err := core.Instance().GetPods("", storkLabel)
```

Comment:
For the `GetPods` API, the namespace parameter given here is `""` it should be the namespace in which stork pods exist.

Reply:
done